pkg/hyperkit/driver.go

```diff
@@ -237,6 +244,10 @@ func (d *Driver) Start() error {
 
 	log.Debugf("Trying to execute %s", h.CmdLine)
 
+	if !d.VMNet {
+		return nil
+	}
+
 	getIP := func() error {
 		st, err := d.GetState()
```

Comment:
getIP() also waits until the hyperkit process is running, would be better not to lose this (though this is also gone in the libvirt machine driver).

Reply:
`GetState` error is not a `RetriableError`. Since it is called right after start, I guess it dosn't catch much errors. Should I convert it to a RetriableError ?

Reply:
what is the difference in behaviour?

Reply:
The wait loop is on crc side now, not in the driver anymore.